src/main/webapp/app/exercises/programming/manage/grading/charts/sca-category-distribution-chart.component.ts

```diff
@@ -38,7 +38,7 @@ enum ScaChartBarTitle {
                     [yAxis]="true"
                     [xAxisTickFormatting]="xAxisFormatting"
                     (select)="onSelect($event)"
-                >
+=                >
```

Comment:
This change seems not intentional.

Reply:
Good catch! Removed the mistake in e1fbbe43e28ea056f1c428c41a72e8deed2f16d6